src/main/java/org/opensearch/flowframework/util/ParseUtils.java

```diff
@@ -385,4 +401,17 @@ public static String parseArbitraryStringToObjectMapToString(Map<String, Object>
         String mappedString = mapper.writeValueAsString(map);
         return mappedString;
     }
+
+    /**
+     * Generates a String to String map based on a Json File
+     * @param path file path
+     * @return instance of the string
+     * @throws JsonProcessingException JsonProcessingException from Jackson for issues processing map
+     */
+    public static Map<String, String> parseJsonFileToStringToStringMap(String path) throws IOException {
+        ObjectMapper mapper = new ObjectMapper();
```

Comment:
nit: consider reusing a single ObjectMapper instance instead of creating a new one each time the method is called

Reply:
moved as a static final variable